build/package/windows_notarize.sh

```diff
@@ -38,7 +38,7 @@ if [[ -f "$EXE_FILE" && -f "$MSI_FILE" ]]; then
 		-v "$(pwd):$(pwd)" \
 		-w "$(pwd)" \
 		artifactory.corp.mongodb.com/release-tools-container-registry-local/garasign-jsign \
-		/bin/bash -c "jsign --tsaurl http://timestamp.digicert.com -a mongo-authenticode-2021 \"$EXE_FILE\""
+		/bin/bash -c "jsign --tsaurl http://timestamp.digicert.com -a ${AUTHENTICODE_KEY_NAME} \"$EXE_FILE\""
```

Comment:
CI is failing because `AUTHENTICODE_KEY_NAME` is present on host but not on container, perhaps update `.env` file

Reply:
also make sure to add quotes